### PR TITLE
Removed partial jquery dep

### PR DESCRIPTION
**Pull request by dshubhadeep**
Removed query dep. in all files except  [list.js](https://github.com/Presto412/CryptoFS/blob/master/src/public/js/list.js).

### CODE REVIEW

**Comment by chris--jones** [comment]
Why not list.js?

File: src/public/js/util.js [hunk 1]
```diff
@@ -26,4 +26,24 @@ const downloadBlob = function (data, fileName, mimeType) {
...
+  });
+}
+
+/**
+ * 
+ * @param {string} selector 
+ */
+const showElement = function (selector) {
+  document.querySelectorAll(selector).forEach(el => {
+    el.style.display = 'block';
```
Comment (chris--jones) [inline src/public/js/util.js:45]: This is not ideal as not all elements may be block-level, instead set the display style property to an empty string.

**Comment by dshubhadeep** [comment]
> Why not list.js?

I had to confirm with @Presto412 on how to replace the ajax function. I'll make the changes shortly.

**Comment by chris--jones** [comment]
> 
> 
> > Why not list.js?
> 
> I had to confirm with @Presto412 on how to replace the ajax function. I'll make the changes shortly.

You can use fetch (https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)
I have used similar in my PR for file encryption #15

**Comment by Presto412** [comment]
@dshubhadeep just merged some changes related to file encryption, could you please resolve conflicts and run 
```sh
npx webpack  --mode=production
```
Thanks

**Comment by dshubhadeep** [comment]
> @dshubhadeep just merged some changes related to file encryption, could you please resolve conflicts and run
> 
> ```shell
> npx webpack  --mode=production
> ```
> 
> Thanks

Done. Please check once

**Comment by Presto412** [comment]
@dshubhadeep Doesn't seem to work, not able to expand Login tab and no message shown in the files tab as well. Please check

**Review by chris--jones — CHANGES_REQUESTED** [review]

**Review by Presto412 — APPROVED** [review]

**Pull request merged**